lib/saml/user.rb

```diff
@@ -30,7 +34,31 @@ def authn_context
     private
 
     def serializable_attributes
-      @decorated.send(:serializable_attributes) + %i(authn_context last_signed_in)
+      @decorated.serializable_attributes + %i(authn_context last_signed_in)
+    end
+
+    # see warnings
+    def log_warnings_to_sentry!
+      warning_context = {
+        authn_context: authn_context || 'nil ~= idme',
```

Comment:
This is going to be `{dslogon|mhv|nil ~= idme}`. But won't tell us if we're getting this on the multifactor or identity_proof endpoints. Wondering if it would be helpful to include `REXML::XPath.first(saml_response.decrypted_document, '//saml:AuthnContextClassRef')&.text` in the sentry context so we can distinguish between the callback-triggering operations.

Reply:
We could probably deduce whether it's identity_proof based on the value of `loa`, but if we're already in here, might as well record the exact value coming back in the saml response.

Reply:
good point, do you think we should just use that instead of the stupid authn_context method?